### ISSUE TITLE
Auto-approve the PR

### ISSUE THREAD

**Issue by plavoie-nms** [shayki5/azure-devops-create-pr-task#189]
We have a policy of requiring one reviewer per PR, and the release should be able to auto-approve and auto-complete the PR. The bypass option does not work because it bypasses the build validation too, and it should not.

Is it possible for the task to approve the PR ?

**Comment by shayki5** [comment]
So do you want only bypass the review but not the build?

**Comment by plavoie-nms** [comment]
Yes, to meet the minimum reviewer count, the pipeline needs to approve its own PR.

**Comment by shayki5** [comment]
I think it's impossible :(
Or auto-complete and you need to approve manually or bypass all the policies.
